### ISSUE TITLE
Support Graphs.jl's generators for common graphs

### ISSUE THREAD

**Issue by mroavi** [JuliaGraphs/MetaGraphsNext.jl#28]
Graphs.jl provides these [generators for common graphs](https://juliagraphs.org/Graphs.jl/dev/core_functions/simplegraphs_generators/).

As far as I can tell, these cannot be used conveniently with MetaGraphsNext.jl yet:

```julia
julia> MetaGraphsNext.MetaGraph(path_graph(5), default_weight = 3.0)

 Warning: Constructing a MetaGraph with a nonempty underlying graph is not advised.
└ @ MetaGraphsNext ~/.julia/dev/MetaGraphsNext/src/metagraph.jl:55
Meta graph based on a {5, 4} undirected simple Int64 graph with vertex labels of type Symbol, vertex metadata of type Nothing, edge metadata of type Nothing, graph metadata given by nothing, and default weight 3.0
```

Would be nice to add support for these.

**Comment by gdalle** [comment]
Related to #37

**Comment by gdalle** [comment]
Solved by the new bulk constructor in #45
